### DISCUSSION TITLE
Upgrade version of solid of js-benchmark?

### DISCUSSION THREAD

**Discussion by zhaoyao91** [solidjs/solid#508] [Ideas]
Congradulations to solid-js v1.0.0! 福

Will the [benchmark-js](https://krausest.github.io/js-framework-benchmark/index.html) for it be upgrade to v1 to reflect its recent performance? It's still v0.20.0, which is an old version.

**Comment by ryansolid** [comment]
He only does those updates when there is a new chrome version. 1.0 is already in the unofficial results. https://krausest.github.io/js-framework-benchmark/current.html

That's an rc from about a week ago. He also gave it a full run for comparison. So about as good as you get for a non-official result.
